src/main/java/com/google/sps/data/Drink.java

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.*;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Drink implements Comparable<Drink> {
+
+  private final String name;
+  private final String storeID;
+  public double avgRating;
+  public double numRatings;
+  Entity drinkEntity;
+
+  Drink(String name, String storeID, double avgRating, double numRatings, Entity drinkEntity) {
+    this.name = name;
+    this.storeID = storeID;
+    this.avgRating = avgRating;
+    this.numRatings = numRatings;
+    this.drinkEntity = drinkEntity;
+  }
+
+  @Override
+  public int compareTo(Drink d) {
+    return new Double(avgRating).compareTo(d.avgRating);
+  }
+
+  @Override
+  public String toString() {
+    return String.valueOf(avgRating);
+  }
+
+  public static List<Drink> searchForDrinkByRating(String name) {
+    List<Drink> drinks = DrinkDAO.getDrinksByName(name);
+    Collections.sort(drinks, Collections.reverseOrder());
+    return drinks;
+  }
+
+  public static Set<Drink> searchForDrink(String name) {
+    List<Drink> drinks = DrinkDAO.getDrinksByName(name);
+    Set<Drink> drinksSet = new HashSet<Drink>();
+    for (Drink drink : drinks) {
+      drinksSet.add(drink);
+    }
+    return drinksSet;
+  }
+
+  public double updateAverageRating(double newRating) {
+    this.avgRating = ((this.avgRating * this.numRatings) + newRating) / (this.numRatings + 1);
+    this.numRatings++;
+    DrinkDAO.updateEntity(this.drinkEntity, this.numRatings, this.avgRating);
+
+    return this.avgRating;
+  }
+
+  public String getName() {
+    return this.name;
+  }
+
+  public String getStore() {
+    return this.storeID;
+  }
+
+  public double getNumRatings() {
+    return this.numRatings;
+  }
+
+  public double getRating() {
+    return this.avgRating;
+  }
+
+  public double roundToOneDecimalPlace(double num) {
```

Comment:
It's fine if you missed it. It's being used for display purposes only, at a later date it might be deleted and replaced with the use of String.format() but as it stands right now I can't think of a reason to replace it.

Reply:
If it's only used for display then use String.format

Reply:
ok, follow up on this in future PRs